packages/autocertifier/src/ChallengeManager.ts

```diff
@@ -0,0 +1,4 @@
+export interface ChallengeManager {
+    createChallenge(name: string, value: string): Promise<void>
+    deleteChallenge(name: string): Promise<void>
```

Comment:
Is this an identifier, or a descriptive name? Maybe rename to `id` if it an identifier

Reply:
renamed to fqdn